speech/unit_tests/test_client.py

```diff
@@ -15,16 +15,9 @@
 import unittest
 
 
-class TestClient(unittest.TestCase):
-    SAMPLE_RATE = 16000
-    HINTS = ['hi']
-    AUDIO_SOURCE_URI = 'gs://sample-bucket/sample-recording.flac'
-    AUDIO_CONTENT = '/9j/4QNURXhpZgAASUkq'
-
-    @staticmethod
-    def _make_result(alternatives):
-        from google.cloud.grpc.speech.v1beta1 import cloud_speech_pb2
-
+def _make_result(alternatives=None):
+    from google.cloud.grpc.speech.v1beta1 import cloud_speech_pb2
+    if alternatives:
```

Comment:
Always check for `None` instead of checking false-y:

```python
if alternatives is not None:
    ...
```

See [PEP8][1]:

> Comparisons to singletons like `None` should always be done with `is` or `is not`, never the equality operators.
>
> Also, beware of writing `if x` when you really mean `if x is not None`

[1]: https://www.python.org/dev/peps/pep-0008/#programming-recommendations